modules/User Admin/user_manage.php

```diff
@@ -48,14 +46,11 @@
         ->sortBy(['surname', 'preferredName'])
         ->fromPOST();
 
-    echo '<h2>';
-    echo __('Search');
-    echo '</h2>';
-    
-    $form = Form::create('filter', $_SESSION[$guid]['absoluteURL'].'/index.php', 'get');
+    $form = Form::create('filter', $gibbon->session->get('absoluteURL').'/index.php', 'get');
+    $form->setTitle(__('Search'));
     $form->setClass('noIntBorder fullWidth');
 
-    $form->addHiddenValue('q', '/modules/'.$_SESSION[$guid]['module'].'/user_manage.php');
+    $form->addHiddenValue('q', '/modules/'.$gibbon->session->get('module').'/user_manage.php');
```

Comment:
As a note, we're moving to $session vs $gibbon->session, feel free to use this shorter syntax in the future. They'll both work for the next whole version, but ideally at some point we'll replace them all (and phase out the $gibbon object, which wont be needed anymore).